src/Configurator/DockerComposeConfigurator.php

```diff
@@ -111,34 +111,12 @@ public static function shouldConfigureDockerRecipe(Composer $composer, IOInterfa
             return false;
         }
 
-        $warning = $io->isInteractive() ? 'WARNING' : 'IGNORING';
-        $io->writeError(sprintf('  - <warning> %s </> %s', $warning, $recipe->getFormattedOrigin()));
-        $question = '    The recipe for this package contains some Docker configuration.
-
-    This may create/update <comment>docker-compose.yml</comment> or update <comment>Dockerfile</comment> (if it exists).
-
-    Do you want to include Docker configuration from recipes?
-    [<comment>y</>] Yes
-    [<comment>n</>] No
-    [<comment>p</>] Yes permanently, never ask again for this project
-    [<comment>x</>] No permanently, never ask again for this project
-    (defaults to <comment>y</>): ';
-        $answer = $io->askAndValidate(
-            $question,
-            function ($value) {
-                if (null === $value) {
-                    return 'y';
-                }
-                $value = strtolower($value[0]);
-                if (!\in_array($value, ['y', 'n', 'p', 'x'], true)) {
-                    throw new \InvalidArgumentException('Invalid choice.');
-                }
+        if ($_SERVER['DOCKER_SUPPORT'] ?? false) {
```

Comment:
let's use `filter_var(..., \FILTER_VALIDATE_BOOLEAN)` and allow enabling or disabling via the env var?

Reply:
what about SYMFONY_DOCKER for the name?

Reply:
Re: enabling/disabling, we want Docker disables by default, and enable it when the env var is "set". We don't really care about the value. If it's false-ish, it's like not having it.

Reply:
Name updated.

> Re: enabling/disabling, we want Docker disables by default, and enable it when the env var is "set". We don't really care about the value. If it's false-ish, it's like not having it.

And so `SYMFONY_DOCKER=0` would work if you, for some reason, wanted to use this env var but explicitly set it to false.

Reply:
it's strange to be able to SYMFONY_DOCKER=0 to enable it

It's easy cheap to implement better now isn't it?

Reply:
SYMFONY_DOCKER=0 must disable it, that's what I mentioned before.

Reply:
oh, right about SYMFONY_DOCKER=0
my point stands for SYMFONY_DOCKER=false :)
using `filter_var(..., \FILTER_VALIDATE_BOOLEAN)` is the way to check boolean envvars (or php.ini settings) in Symfony to me

Reply:
So `SYMFONY_DOCKER=1` to permanently enable Docker support and `SYMFONY_DOCKER=0` to permanently disable it (without being asked)?

+1 to use `filter_var()`.